From 5a615cf46fe39b5974e5a758cf8d8b0c30ce6ae3 Mon Sep 17 00:00:00 2001
From: "JianFeng,Zhou" <jianfeng.zhou@intel.com>
Date: Fri, 31 Jul 2020 08:30:02 +0800
Subject: [PATCH] SDE/FRP:add vsock sepolicy for recovery

enable recovery notify host application by vsock

Change-Id: I17487362547431e8d7d23ac91c79923976542720
Tracked-On: OAM-92884
Signed-off-by: JianFeng,Zhou <jianfeng.zhou@intel.com>
Signed-off-by: Xiao He <xiao.he@intel.com>
---
 vendor/recovery.te | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/vendor/recovery.te b/vendor/recovery.te
index 8e4d19c..00b32b9 100644
--- a/vendor/recovery.te
+++ b/vendor/recovery.te
@@ -1,4 +1,9 @@
 recovery_only(`
   allow recovery gpu_device:dir  search;
   allow recovery sysfs_backlight_thermal:dir  search;
+  allow recovery recovery:vsock_socket create;
+  allow recovery recovery:vsock_socket connect;
+  allow recovery recovery:vsock_socket write;
+  allow recovery recovery:vsock_socket read;
+  allow recovery recovery:capability net_admin;
 ')
--
2.17.1

